codes/quantum/qubits/movassagh_ouyang

```diff
@@ -0,0 +1,86 @@
+  # encoders: 
+    # - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+    # - 'Measurement-based preparation ... with ancilla overhead of ... .'
+    # - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+    # - 'Universal gate set achieved by either additional ... gate.'
+    # - 'Magic-state distillation protocols'
+    # - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  # decoders:
+    # - 'Details about how syndrome measurements are done; discuss overhead, if applicable.'
+    # - 'MWPM decoding algorithm \cite{doi:good-paper} with ... overhead.'
+    # - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+    # - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+    # - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+    # - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+    # - 'Pieceable fault tolerance.'
+
+  # threshold: 
+    # - '2\% error-correction threshold  ... with perfect ... decoder \cite{arxiv:another-paper}.'
+    # - '0.3\% error-correction threshold ... with noisy MWPM decoder \cite{doi:good-paper}.'
+    # - '\(10^{-5}\) computational threshold using concatenated scheme under ... noise with overhead of ... '
+    # - 'Asymptotic lower bound of ... on threshold with ... noise with ... overhead.'
+    # - 'Thresholds using magic-state distillation protocols assuming perfect Clifford gates.'
+  
+  # # Propose a field that you feel is applicable to a large and interesting class of codes
+  # #some_other_field:
+  # # - 'detail'
+
+# realizations:
+  # # List and explain the different "domains" of realizations in list items.
+  # # Group closely related realizations into a single item. It's OK to have a
+  # # single list item. Each item should more or less correspond to what would be
+  # # said in a paragraph.
+  # - 'Code used in DVDs \cite{doi:####...}, 5G, etc.'
+  # - 'Realized in trapped-ion quantum devices \cite{arXiv:####.#####}, etc.'
+
+notes:
+  - 'If the probabilistic algorithm fails, it is still possible to create an approximate quantum error correcting code.'
+  - 'This framework works for qudits as well, but is more easily stated for qubits.'
+
+relations:
+  parents:
+    - code_id: qecc
+      detail: ''
+
+  cousins:
+    - code_id: css
+      detail: ''
+    - code_id: cws
+      detail: 'The Movassagh-Ouyang codes overlap the CWS (code-word stabilized) codes but neither family is contained in the other.'
+    - code_id: quantum_ldpc
+      detail: 'These codes reside in the ground space of a local Hamiltonian (which could be taken as the definition of a quantum LDPC code).'
+    - code_id: stabilizer
+      detail: 'Many of the codes in this family are stabilizer codes (but not all).'
```

Comment:
- code_id: hamiltonian
      detail: 'Codespace is the ground-state space of a local Hamiltonian.'